src/hooks/useGetSourceParsedTokenAccounts.ts

```diff
@@ -640,18 +640,17 @@ const createNFTParsedTokenAccountFromCovalent = (
   covalent: CovalentData,
   nft_data: CovalentNFTData
 ): NFTParsedTokenAccount => {
+  let amount =
```

Comment:
please set the variable as const instead of let

```suggestion
  const amount =
```